.github/workflows/ibis-backends.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         run: just download-data
 
       - name: install backend data
-        run: just load-data ${{ matrix.backend.name }}
+        run: poetry run python ci/datamgr.py load ${{ matrix.backend.name }}
```

Comment:
What's our preference here?

Reply:
`poetry run` is required in CI, but not for local dev (but even that depends!). I don't know if it's straightforward to make one command work for both. I can look into it, but the short answer for now is: prefer explicit `poetry run` in CI (`ci-check` notwithstanding) and use `just` for local dev.

Reply:
Thanks! Asked because we have a mixture in the ymls.